azure-pipelines.yml

```diff
@@ -163,22 +163,29 @@ stages:
             queue: BuildPool.Server.Amd64.VS2019
         strategy:
           matrix:
-            debug:
-              _BuildConfig: Debug
-              _PublishArgs: /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
-            release:
-              _BuildConfig: Release
-              ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-                _DotNetPublishToBlobFeed: true
-                _PublishArgs: /p:PublishToSymbolServer=true
-                  /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
-                  /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
-                  /p:PublishToAzure=true
-                  /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
-                  /p:DotNetArtifactsCategory=$(_DotNetArtifactsCategory)
-                  /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
-                  /p:DotNetPublishBlobFeedUrl=$(_PublishBlobFeedUrl)
-                  /p:DotNetPublishToBlobFeed=$(_DotNetPublishToBlobFeed)
+            ${{ if eq(variables['System.TeamProject'], 'public') }}:
+              debug:
+                _BuildConfig: Debug
+                _PublishArgs: /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
+              release:
+                _BuildConfig: Release
+                _PublishArgs: /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
+            ${{ if ne(variables['System.TeamProject'], 'public') }}:
+              release:
+                _BuildConfig: Release
+                ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
+                  _PublishArgs: /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
```

Comment:
Seems like if someone does a PR build on internal only it should universally turn off publish

Reply:
What does `/p:DotNetPublishUsingPipelines=...` do w/o the other global properties in an internal build?

Reply:
probably get ignored.  My point was mostly that it seems some things could just be omitted in some contexts where they're not.

Reply:
It seemed to be causing the debug builds to push their packages to PackageArtifacts & upload a manifest w/ the same names as the release builds

Reply:
That was the motivation behind the comment that started this thread;  I thought you'd want all the publishing turned off entirely for public and Debug.